shenyu-admin/pom.xml

```diff
@@ -49,6 +49,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.shenyu</groupId>
+            <artifactId>shenyu-plugin-base</artifactId>
+            <version>${project.version}</version>
```

Comment:
I think shenyu-admin should not denpend on shenyu plugin.

Reply:
done,It has been changed to use compile-time dependency to verify the jar package.